<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFriendsWindow</class>
 <widget class="QWidget" name="AddFriendsWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>添加好友/聊天室</string>
  </property>
  <widget class="QPushButton" name="search_pushbutton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>200</y>
     <width>120</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>..黑体UI-日本语</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_search_line_edit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>420</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="input_info_index">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>请输入要添加的</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="type_combo_box">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>好友</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>聊天室</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="secretButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ff0000</string>
   </property>
   <property name="text">
    <string>结交志同道合的朋友</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
